src/components/Formulários/Formulario.vue

```diff
@@ -0,0 +1,369 @@
+<template>
+    <div class="main">
+        <section>
+            <div class="welcome">
+                <h1>Seja bem-vindo.</h1>
+                <p>
+                    Para abrir sua conta na <span>Warren</span>, basta você preencher um pequeno cadastro com suas informações. Solicitamos que preencha suas credenciais com atenção.
+                </p>
+            </div>
+            <div class="form">
+                <form>
```

Comment:
poderia remover a div, e colocar a class form na tag  form. 
```suggestion
                <form class="form">
```